tests/integration/tui/screenshots/test_tui_basics.test-rakiura-help.html

```diff
@@ -15,26 +15,26 @@
 <span style="color:#000000;background:#e5e5e5">          </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">                                                      </span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">          </span>
 <span style="color:#000000;background:#e5e5e5">          </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">                           _,@@@@@@.                  </span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">          </span>
 <span style="color:#000000;background:#e5e5e5">          </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">                         &lt;=@@@, `@@@@@.               </span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">          </span>
-<span style="color:#000000;background:#e5e5e5">          </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">                            `-@@@@@@@@@@@'            </span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">          </span>
+<span style="color:#000000;background:#e5e5e5">          </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">                            `-@@@@@@@@@@@&#x27;            </span><span style="color:#000000;background:#e5e5e5">  </span><span style="color:#000000;background:#e5e5e5">│</span><span style="color:#000000;background:#e5e5e5">          </span>
```

Comment:
Looks like they've changed the HTML output to escape quote charters.

Seems legit, tested as working.